modules/op-about-workspace.adoc

```diff
@@ -43,13 +43,18 @@ spec:
   tasks: <2>
   - name: build-image
     taskRef:
-      name: buildah
-      kind: ClusterTask
-    params:
-    - name: TLSVERIFY
-      value: "false"
-    - name: IMAGE
-      value: $(params.IMAGE)
+      resolver: cluster
+      params:
+        - name: kind
+          value: task
+        - name: name
+          value: buildah
+        - name: namespace
+          value: openshift-pipelines
+        - name: TLSVERIFY
```

Comment:
parameters `TLSVERIFY` and `IMAGE` will stay where they were. First three are params for resolved, the last two are params for the task itself.

Reply:
need to double check I understand this correctly - if the pipeline or task parameters are specified in a different block from the resolver parameters, this is a problem in resolver docs

Reply:
verified, changing